src/promptflow/promptflow/executor/_service/apis/execution.py

```diff
@@ -83,6 +84,7 @@ def flow_test(request: FlowExecutionRequest):
     # resolve environment variables
     set_environment_variables(request)
     enable_async_execution()
+    setup_exporter_from_environ()
```

Comment:
I guess we can setup this when Flow.create

Reply:
Seems like we will invoke `flow_test` in new process.
And we need environment variable to setup exporter.
Not sure what's the meaning for `Flow.create`

Reply:
Offline synced, invoke setup as creating executor is better.
If so, we could skip this operation in many places.